EIPS/eip-5656.md

```diff
@@ -0,0 +1,104 @@
+---
+eip: 5656
+title: Memory copying instruction
+description: An efficient EVM instruction for copying memory areas
+author: Alex Beregszaszi (@axic), Paul Dworzanski (@poemm), Jared Wasinger (@jwasinger), Casey Detrio (@cdetrio), Pawel Bylica (@chfast), Charles Cooper (@charles-cooper)
+discussions-to: https://ethereum-magicians.org/t/eip-5656-mcopy-instruction/10890
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-02-01
+---
+
+## Abstract
+
+Provide an efficient EVM instruction for copying memory areas.
+
+## Motivation
+
+Memory copying is a basic operation, yet implementing it on the EVM comes with overhead.
```

Comment:
```suggestion
Memory copying is a frequent operation, yet implementing it on the EVM comes with significant overhead.
```
Basic != Frequently Used
Significant: It's going to come with overhead, the point is that right now it's too much overhead.